bestie/server/testresult.go

```diff
@@ -71,6 +55,40 @@ func identifyStream(bazelBuildEvent bes.BuildEvent, streamId *build.StreamId) ba
 	return stream
 }
 
+// Read the contents of a bytestream file.
+func readBytestreamFile(fileName, bytestreamUri string) ([]byte, error) {
```

Comment:
Not sure if this is feasible, but will take some investigation. I don't have time to pursue this now, so I've opened [INFRA-504](https://enfabrica.atlassian.net/browse/INFRA-504) to track it. Feel free to reassign, as appropriate.

Reply:
Ok - this is going to be very important to follow up on - we are planning on sending builds that are going to generate giant output.zip files (GBs) and as written right now this will try and load it into RAM and bestie will probably fall over